### PR TITLE
Tests on php 7.3

### PR DESCRIPTION
**Pull request by samnela**

### CODE REVIEW

**Comment by kelunik** [comment]
Please keep 7.0.

**Comment by samnela** [comment]
Just update , `--ignore-platform-reqs` causes the problem with php 7.0 because it installs symfony/console 4.2 (> = php 7.1) which is not compatible

File: .travis.yml [hunk 1]
```diff
@@ -15,8 +16,9 @@ matrix:
 
 before_script:
   # --ignore-platform-reqs, because https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/2722
-  - composer update -n --prefer-dist --ignore-platform-reqs
-  - composer require satooshi/php-coveralls dev-master --ignore-platform-reqs
+  - if [[ "${TRAVIS_PHP_VERSION}" != "7.0" ]]; then composer update -n --prefer-dist --ignore-platform-reqs; fi
+  - if [[ "${TRAVIS_PHP_VERSION}" == "7.0" ]]; then composer update -n --prefer-dist; fi
+  - composer require --dev php-coveralls/php-coveralls ^2.0.0 --ignore-platform-reqs
```
Comment (samnela) [inline .travis.yml:21]: maybe we should put this dependency in autoload-dev

Comment (kelunik) [reply .travis.yml:21]: Either that or change it to the PHAR, [as Amp does it](https://github.com/amphp/amp/blob/216315ee5c59be6847af3391f11f04404d1ce659/.travis.yml).

**Comment by kelunik** [comment]
Thanks!

**Pull request merged**